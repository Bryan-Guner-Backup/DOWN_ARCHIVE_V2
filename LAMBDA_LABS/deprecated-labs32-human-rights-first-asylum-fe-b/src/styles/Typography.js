import { createGlobalStyle } from 'styled-components';

const Typography = createGlobalStyle`
    html {
        font-family: Helvetica, Arial, sans-serif;
        color: var(--font);
        font-size: 62.5%;
    }
`;

export default Typography;
