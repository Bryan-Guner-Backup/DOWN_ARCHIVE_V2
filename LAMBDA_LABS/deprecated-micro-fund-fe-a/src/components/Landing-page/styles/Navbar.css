.nav{
    width: 100%;
    background-color: #1e0c86;
    display: flex;
    flex-direction: column;
    
}


nav{
    display: flex;
    justify-content: center;
    background-color: #1b0a78;
    padding: 10px
}

#img{
    margin: 0 auto;
}
img{
    width: 200px;
    margin: 0 auto;
}
nav a{
    text-decoration: none;
    color: white;
    font-size: 20px;

}

