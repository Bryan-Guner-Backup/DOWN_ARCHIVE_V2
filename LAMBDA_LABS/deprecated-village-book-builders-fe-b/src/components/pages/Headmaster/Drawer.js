import React from 'react';
import { Link } from 'react-router-dom';

function HeadmasterNav() {
  return (
    <div>
      <Link to="/mentor-pairings">Mentor Pairings</Link>
      <Link to="/mentor-advisor">Mentor Advisor</Link>
      <Link to="/school-village">School/Village</Link>
      <Link to="/library">Library</Link>
    </div>
  );
}

export default HeadmasterNav;
