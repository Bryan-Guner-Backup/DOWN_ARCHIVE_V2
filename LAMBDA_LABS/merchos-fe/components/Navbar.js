import { useState } from "react";
import Link from "next/link";
import styled from "styled-components";
import { IoMdArrowBack, IoMdContact } from "react-icons/io";
import Router from "next/router";
import { useDispatch } from "react-redux";
import { authorizeUser } from "../store/actions/userAuth/userAuthActions";

const Wrapper = styled.section`
  z-index: 100;
  display: flex;
  justify-content: space-between;
  align-content: center;
  align-items: center;

  width: auto;
  height: 10%;
  padding-top: 1%;
  padding-left: 2%;
  padding-right: 6%;

  background: #f3f3ff;

  .log {
    display: none;
    transition: 0.5s;
  }
  .dropmenu {
    display: block;
    position: absolute;
  }
`;
const ProfileBtn = styled.button`
  width: 145px;
  height: 50px;

  font-family: "Roboto", sans-serif;
  font-style: normal;
  font-weight: normal;
  font-size: 2rem;
  line-height: 29px;
  display: flex;
  align-items: center;
  justify-content: space-around;

  color: #565656;
  cursor: pointer;

  background: #f3f3ff;
  box-shadow: -2px -2px 6px 2px #fff, 2px 2px 6px 2px #787878;
  border-radius: 55px;
`;
const Title = styled.h1`
  font-family: "Roboto", sans-serif;
  font-style: light;
  // font-weight: 10;
  font-size: 50px;
  line-height: 73px;
  display: flex;
  align-items: center;
  text-align: center;
  justify-content: center;

  color: #565656;
`;
const BackBtn = styled.button`
  width: 145px;
  height: 50px;

  background: #f3f3ff;
  box-shadow: -2px -2px 6px 2px #fff, 2px 2px 6px 2px #787878;
  border-radius: 55px;
  cursor: pointer;

  font-family: "'Roboto', sans-serif";

  font-size: 2rem;

  display: flex;
  align-items: center;
  text-align: center;
  justify-content: center;

  color: #565656;
`;
const DropLogout = styled.h1`
  display: none;
  width: 150px;
  height: 40px;

  font-family: "'Roboto', sans-serif";
  font-style: normal;
  font-weight: normal;
  font-size: 2rem;
  line-height: 29px;
  display: flex;
  align-items: center;
  text-align: center;
  justify-content: center;
  cursor: pointer;

  color: #871a04;

  background: #f3f3ff;
  box-shadow: -2px -2px 6px 2px #fff, 2px 2px 6px 2px #787878;
  border-radius: 15px;
  z-index: -8;
`;

const Navbar = props => {
  const dispatch = useDispatch();
  const logout = () => {
    dispatch(authorizeUser("Log Out"));

    Router.push("/");
    console.log("logging out");
  };

  const [isHovering, setIsHovering] = useState(false);
  const hoveringState = () => {
    setIsHovering(!isHovering);
  };

  return (
    <Wrapper>
      <BackBtn onClick={()=>window.history.back()}>
        <IoMdArrowBack size="4rem" color="#565656" />
        Back
      </BackBtn>
      <Title>Back Office</Title>
      {/* <ProfileBtn> */}
      {/* <ProfileBtn onClick={Hidden}> */}

      {/* <label for="select">Hello</label>
                <div className="select"> */}
      <ul
        className="prof"
        onMouseEnter={hoveringState}
        onMouseLeave={hoveringState}
      >
        <li>
          <ProfileBtn>
            {" "}
            Account
            <IoMdContact size="4rem" color="#565656" />
          </ProfileBtn>
        </li>
        <li className={`log ${isHovering === true ? "dropmenu" : ""}`}>
          <DropLogout type="button" id="hide" value="1" onClick={logout}>
            Sign Out
          </DropLogout>
        </li>
      </ul>
      {/* </div> */}

      {/* </ProfileBtn> */}
    </Wrapper>
  );
};

export default Navbar;
