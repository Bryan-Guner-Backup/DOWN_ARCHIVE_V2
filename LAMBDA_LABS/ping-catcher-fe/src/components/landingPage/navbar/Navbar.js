import React from "react";

import "./nav.css";

const Navbar = () => {
  return (
    <nav className="nav">
      <img src="/images/Ping_Catcher.png" className="logo" alt="application" />
      {/* <ul className="nav-links">
        <li>About</li>
        <li>Contact Us</li>
      </ul> */}
    </nav>
  );
};

export default Navbar;
