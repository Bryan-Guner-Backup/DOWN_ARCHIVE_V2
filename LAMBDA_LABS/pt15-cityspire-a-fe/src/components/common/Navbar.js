import React, { useState } from 'react';
import styled from 'styled-components';
import { useOktaAuth } from '@okta/okta-react';
import { ButtonElement } from './';
import { Link } from 'react-router-dom';
import { Layout, Typography, Menu, Button } from 'antd';

const Navbar = () => {
  const [isOpen, setIsOpen] = useState(false);
  const { authState, authService } = useOktaAuth();
  return (
    <Nav>
      <Logo>
        <Link to="/">
          City<span>Spire</span>
        </Link>
      </Logo>
      <HamburgerMenu onClick={() => setIsOpen(!isOpen)}>
        <span />
        <span />
        <span />
      </HamburgerMenu>
      <MenuElemnet
        isOpen={isOpen}
        theme="dark"
        mode="horizontal"
        defaultSelectedKeys={['2']}
      >
        <MenuLink>
          <Link to="/profile-list">My Profile</Link>
        </MenuLink>
        <Button
          type="primary"
          handleClick={() => authService.logout()}
          buttonText="Logout"
        >
          LogOut
        </Button>
      </MenuElemnet>
    </Nav>
  );
};

export default Navbar;

const Nav = styled.div`
  padding: 0 1rem;
  display: flex;
  justify-content: space-between;
  align-items: center;
  flex-wrap: wrap;
  background: RGB(219, 220, 220);
  background: #fff;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
`;

const Logo = styled.a`
  padding: 1rem 0;
  color: RGB(252, 69, 67);
  text-decoration: none;
  font-weight: 800;
  font-size: 1.7rem;

  span {
    font-weight: 300;
    font-size: 1.3rem;
  }
`;

const MenuLink = styled(Link)`
  padding: 1rem 2rem;
  cursor: pointer;
  text-align: center;
  text-decoration: none;
`;

const MenuElemnet = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: relative;
  @media (max-width: 768px) {
    overflow: hidden;
    flex-direction: column;
    max-height: ${({ isOpen }) => (isOpen ? '350px' : '0')};
    transition: max-height 0.3s ease-in;
    width: 100%;
    justify-content: space-around;
    padding-bottom: 20px;
  }
`;

const HamburgerMenu = styled.div`
  display: flex;
  flex-direction: column;
  cursor: pointer;
  span {
    height: 2px;
    width: 25px;
    background: #1890ff;
    margin-bottom: 4px;
    border-radius: 5px;
  }
  @media (min-width: 768px) {
    display: none;
  }
`;
