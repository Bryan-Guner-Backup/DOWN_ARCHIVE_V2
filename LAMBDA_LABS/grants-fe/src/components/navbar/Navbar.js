import React, { useState } from "react";
import { useSelector } from "react-redux";
import { Link, useHistory } from "react-router-dom";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import IconButton from "@material-ui/core/IconButton";
import Button from "@material-ui/core/Button";
import InputBase from "@material-ui/core/InputBase";
import Badge from "@material-ui/core/Badge";
import SearchIcon from "@material-ui/icons/Search";
import AccountCircle from "@material-ui/icons/AccountCircle";
import ChatIcon from "@material-ui/icons/Chat";
import FavoriteIcon from "@material-ui/icons/Favorite";
import FavoriteBorderIcon from "@material-ui/icons/FavoriteBorder";

import MoreIcon from "@material-ui/icons/MoreVert";

import { useStyles } from "./Navbar.styles";
import MobileMenu from "./MobileMenu";
import MenuComponent from "./MenuComponent";

export default function PrimarySearchAppBar() {
  const classes = useStyles();
  const history = useHistory();
  const [anchorEl, setAnchorEl] = useState(null);
  const [mobileMoreAnchorEl, setMobileMoreAnchorEl] = useState(null);

  const [searchTerm, setSearchTerm] = useState("");

  const favorites = useSelector((state) => state.favorites.favoritesCount);
  // should ultimately come from global state
  const [chats] = useState(5);

  const isMenuOpen = Boolean(anchorEl);

  const handleProfileMenuOpen = (event) => {
    setAnchorEl(event.currentTarget);
  };

  const handleMobileMenuClose = () => {
    setMobileMoreAnchorEl(null);
  };

  const handleMenuClose = () => {
    setAnchorEl(null);
    handleMobileMenuClose();
  };

  const handleMobileMenuOpen = (event) => {
    setMobileMoreAnchorEl(event.currentTarget);
  };

  const handleChange = (e) => {
    setSearchTerm(e.target.value);
  };

  const handleSubmit = () => {
    setSearchTerm("");
  };

  const menuId = "primary-search-account-menu";

  const mobileMenuId = "primary-search-account-menu-mobile";

  return (
    <div className={classes.grow}>
      <AppBar position="static">
        <Toolbar>
          <Button component={Link} to="/Homepage" className={classes.title}>
            Granted
          </Button>

          {/* searchbar was originally here */}
          <div className={classes.grow} />
          <div className={classes.search} onClick={handleSubmit}>
            <div className={classes.searchIcon}>
              <SearchIcon />
            </div>
            <InputBase
              data-testid="search"
              placeholder="Searchâ€¦"
              value={searchTerm}
              classes={{
                root: classes.inputRoot,
                input: classes.inputInput,
              }}
              onChange={(e) => handleChange(e)}
              inputProps={{ "aria-label": "search" }}
            />
          </div>
          <div className={classes.sectionDesktop}>
            <IconButton
              aria-label={`show ${favorites} new Favorite`}
              color="inherit"
            >
              <Badge
                badgeContent={favorites}
                color="secondary"
                data-testid="notificationBadge"
              >
                {/* if user has favorited a grant, the icon renders as a filled in heart with the number of favorties, if the user has not faved agrant, the icon is a heart border  */}
                {favorites === 0 ? (
                  <FavoriteBorderIcon />
                ) : (
                  <FavoriteIcon
                    onClick={() => history.push("/writer-favorites")}
                  />
                )}
              </Badge>
            </IconButton>
            <IconButton aria-label={`show ${chats} new Chats`} color="inherit">
              <Badge
                data-testid="messageBadge"
                badgeContent={chats}
                color="secondary"
              >
                <ChatIcon />
              </Badge>
            </IconButton>
            <IconButton
              edge="end"
              aria-label="account of current user"
              aria-controls={menuId}
              aria-haspopup="true"
              onClick={handleProfileMenuOpen}
              color="inherit"
            >
              <AccountCircle />
            </IconButton>
          </div>
          <div className={classes.sectionMobile}>
            <IconButton
              aria-label="show more"
              aria-controls={mobileMenuId}
              aria-haspopup="true"
              onClick={handleMobileMenuOpen}
              color="inherit"
            >
              <MoreIcon />
            </IconButton>
          </div>
        </Toolbar>
      </AppBar>
      <MobileMenu
        mobileMoreAnchorEl={mobileMoreAnchorEl}
        mobileMenuId={mobileMenuId}
        handleMobileMenuClose={handleMobileMenuClose}
        handleProfileMenuOpen={handleProfileMenuOpen}
        chats={chats}
        favorites={favorites}
      />
      <MenuComponent
        anchorEl={anchorEl}
        menuId={menuId}
        isMenuOpen={isMenuOpen}
        handleMenuClose={handleMenuClose}
      />
    </div>
  );
}
