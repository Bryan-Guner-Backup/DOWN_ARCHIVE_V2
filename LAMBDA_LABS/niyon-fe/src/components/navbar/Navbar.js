import React from 'react'
import AppBar from '@material-ui/core/AppBar'
import Button from '@material-ui/core/Button'
import CssBaseline from '@material-ui/core/CssBaseline'
import Drawer from '@material-ui/core/Drawer'
import Hidden from '@material-ui/core/Hidden'
import IconButton from '@material-ui/core/IconButton'
import List from '@material-ui/core/List'
import MenuIcon from '@material-ui/icons/Menu'
import CloseIcon from '@material-ui/icons/Close'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'
import { makeStyles, useTheme } from '@material-ui/core/styles'
import { Link } from 'react-router-dom'

const drawerWidth = 240
const useStyles = makeStyles((theme) => ({
  root: {
    display: 'flex'
  },
  drawer: {
    [theme.breakpoints.up('sm')]: {
      width: drawerWidth,
      flexShrink: 0
    }
  },
  appBar: {
    zIndex: theme.zIndex.drawer + 1
  },
  menuButton: {
    marginRight: theme.spacing(2),
    [theme.breakpoints.up('sm')]: {
      display: 'none'
    }
  },
  toolbar: theme.mixins.toolbar,
  drawerPaper: {
    width: drawerWidth
  },
  content: {
    flexGrow: 1,
    padding: theme.spacing(3)
  },
  closeMenuButton: {
    marginRight: 'auto',
    marginLeft: 0
  },
  buttons: {
    margin: '20% auto 0 auto',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    paddingTop: '10px'
  },
  button: {
    width: '250px',
    borderRadius: '50px',
    height: '50px',
    margin: '15px 0',
    textTransform: 'capitalize'
  },
  list: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'start',
    paddingLeft: '10px'
  },
  menulink: {
    color: 'black',
    textTransform: 'capitalize',
    textDecoration: 'none'
  }
}))

function Navbar (props) {
  const classes = useStyles()
  const theme = useTheme()
  const [mobileOpen, setMobileOpen] = React.useState(false)
  function handleDrawerToggle () {
    setMobileOpen(!mobileOpen)
  }
  const drawer = (
    <div>
      <List className={classes.list}>
        <Link to="/">
          <Button className={classes.menulink}>Home</Button>
        </Link>
        <Link to="/about">
          <Button className={classes.menulink}> About</Button>
        </Link>
      </List>
    </div>
  )
  return (
    <div className={classes.root} data-test="navbar-container">
      <CssBaseline />
      <AppBar position="fixed" className={classes.appBar}>
        <Toolbar>
          <div className="logo"></div>
          <div className="title-and-button">
            <Typography variant="h6" noWrap>
              Niyon
            </Typography>
            <IconButton
              color="inherit"
              aria-label="Open drawer"
              edge="start"
              onClick={handleDrawerToggle}
              className={classes.menuButton}
            >
              <MenuIcon />
            </IconButton>
          </div>
        </Toolbar>
      </AppBar>
      <nav className={classes.drawer}>
        {/* The implementation can be swapped with js to avoid SEO duplication of links. */}
        <Hidden smUp implementation="css">
          <Drawer
            variant="temporary"
            anchor={theme.direction === 'rtl' ? 'right' : 'left'}
            open={mobileOpen}
            onClose={handleDrawerToggle}
            classes={{
              paper: classes.drawerPaper
            }}
            ModalProps={{
              keepMounted: true // Better open performance on mobile.
            }}
          >
            <IconButton
              onClick={handleDrawerToggle}
              className={classes.closeMenuButton}
            >
              <CloseIcon />
            </IconButton>
            {drawer}
          </Drawer>
        </Hidden>
        <Hidden xsDown implementation="css">
          <Drawer
            className={classes.drawer}
            variant="permanent"
            classes={{
              paper: classes.drawerPaper
            }}
          >
            <div className={classes.toolbar} />
            {drawer}
          </Drawer>
        </Hidden>
      </nav>
    </div>
  )
}

export default Navbar
