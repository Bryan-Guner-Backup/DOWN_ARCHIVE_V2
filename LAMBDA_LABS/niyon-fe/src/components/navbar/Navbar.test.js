import React from 'react'
import toJSON from 'enzyme-to-json'
import { shallow } from 'enzyme'
import AppBar from '@material-ui/core/AppBar'
import CssBaseline from '@material-ui/core/CssBaseline'
import Drawer from '@material-ui/core/Drawer'
import Hidden from '@material-ui/core/Hidden'
import IconButton from '@material-ui/core/IconButton'
import MenuIcon from '@material-ui/icons/Menu'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'
import findByTestAttr from '../../tests/utils'
import Navbar from './Navbar'

const setUp = (props = {}, state = null) => {
  const wrapper = shallow(<Navbar />)
  return wrapper
}

describe('<Marketing /> component testing', () => {
  let component
  const setState = jest.fn()
  const useStateSpy = jest.spyOn(React, 'useState')
  useStateSpy.mockImplementation((init) => [init, setState])
  beforeEach(() => {
    component = setUp()
  })

  it('should pass snapshot testing correctly', () => {
    expect(toJSON(component)).toMatchSnapshot()
  })

  it('should render <Marketing /> component correctly', () => {
    const marketingWrapper = findByTestAttr(component, 'navbar-container')
    expect(marketingWrapper.length).toBe(1)
  })

  it('should render  <CssBaseline /> component with no errors', () => {
    expect(component.find(CssBaseline)).toHaveLength(1)
  })

  it('should render <AppBar /> component correctly', () => {
    expect(component.find(AppBar)).toHaveLength(1)
  })
  it('should render <Toolbar /> component correctly', () => {
    expect(component.find(Toolbar)).toHaveLength(1)
  })
  it('should render <IconButton /> component correctly', () => {
    expect(component.find(Toolbar).find(IconButton)).toHaveLength(1)
  })

  it('should render <MenuIcon /> component correctly', () => {
    expect(component.find(MenuIcon)).toHaveLength(1)
  })

  it('should render title content with no errors', () => {
    expect(component.find(Typography)).toHaveLength(1)
    expect(component.find(Typography).text()).toEqual('Niyon')
  })

  it('should render navbar component correctly', () => {
    expect(component.find('.makeStyles-drawer-2').exists()).toBe(true)
  })

  describe('First <Hidden /> component', () => {
    it('should render <Hidden /> with smUp attribute correctly', () => {
      const hiddenWrapper1 = component.find(Hidden).at(0)
      expect(hiddenWrapper1.exists()).toBe(true)
      expect(hiddenWrapper1.props().smUp).toEqual(true)
    })
    it('should render the <Drawer /> component correctly', () => {
      const hiddenWrapper1 = component.find(Hidden).at(0)
      expect(hiddenWrapper1.find(Drawer)).toHaveLength(1)
    })
    it('should render <IconButton /> inside <Drawer /> component correctly', () => {
      const hiddenWrapper1 = component.find(Hidden).at(0)
      expect(hiddenWrapper1.find(Drawer).find(IconButton)).toHaveLength(1)
    })
  })

  describe('Second <Hidden /> component', () => {
    it('should render <Hidden /> with xsDown attribute correctly', () => {
      const hiddenWrapper2 = component.find(Hidden).at(1)
      expect(hiddenWrapper2.exists()).toBe(true)
      expect(hiddenWrapper2.props().xsDown).toEqual(true)
    })
    it('should render the <Drawer /> component correctly', () => {
      const hiddenWrapper2 = component.find(Hidden).at(1)
      expect(hiddenWrapper2.find(Drawer)).toHaveLength(1)
    })
    it('should render a <Div /> inside <Drawer /> component correctly', () => {
      const hiddenWrapper2 = component.find(Hidden).at(1)
      expect(hiddenWrapper2.find('.makeStyles-toolbar-5').exists()).toBe(true)
    })
  })

  describe('Actions in <Navbar /> component', () => {
    it('should toggle the first IconButton', () => {
      const handleDrawerToggle = () => jest.fn()
      component.find(Toolbar)
        .find(IconButton)
        .props()
        .onClick(handleDrawerToggle)
      expect(setState).toHaveBeenCalled()
    })

    it('should toggle the first Drawer', () => {
      const handleDrawerToggle = () => jest.fn()
      component.find(Hidden)
        .at(0)
        .find(Drawer)
        .props()
        .onClose(handleDrawerToggle)
      expect(setState).toHaveBeenCalled()
    })
  })
})
