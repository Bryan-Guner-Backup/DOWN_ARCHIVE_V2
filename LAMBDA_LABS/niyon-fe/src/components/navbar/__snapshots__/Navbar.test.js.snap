// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Marketing /> component testing should pass snapshot testing correctly 1`] = `
<div
  className="makeStyles-root-1"
  data-test="navbar-container"
>
  <WithStyles(CssBaseline) />
  <WithStyles(ForwardRef(AppBar))
    className="makeStyles-appBar-3"
    position="fixed"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <div
        className="logo"
      />
      <div
        className="title-and-button"
      >
        <WithStyles(ForwardRef(Typography))
          noWrap={true}
          variant="h6"
        >
          Niyon
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(IconButton))
          aria-label="Open drawer"
          className="makeStyles-menuButton-4"
          color="inherit"
          edge="start"
          onClick={[Function]}
        >
          <MenuIcon />
        </WithStyles(ForwardRef(IconButton))>
      </div>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
  <nav
    className="makeStyles-drawer-2"
  >
    <Hidden
      implementation="css"
      smUp={true}
    >
      <WithStyles(ForwardRef(Drawer))
        ModalProps={
          Object {
            "keepMounted": true,
          }
        }
        anchor="left"
        classes={
          Object {
            "paper": "makeStyles-drawerPaper-6",
          }
        }
        onClose={[Function]}
        open={false}
        variant="temporary"
      >
        <WithStyles(ForwardRef(IconButton))
          className="makeStyles-closeMenuButton-8"
          onClick={[Function]}
        >
          <CloseIcon />
        </WithStyles(ForwardRef(IconButton))>
        <div>
          <WithStyles(ForwardRef(List))
            className="makeStyles-list-11"
          >
            <Link
              to="/"
            >
              <WithStyles(ForwardRef(Button))
                className="makeStyles-menulink-12"
              >
                Home
              </WithStyles(ForwardRef(Button))>
            </Link>
            <Link
              to="/about"
            >
              <WithStyles(ForwardRef(Button))
                className="makeStyles-menulink-12"
              >
                 About
              </WithStyles(ForwardRef(Button))>
            </Link>
          </WithStyles(ForwardRef(List))>
        </div>
      </WithStyles(ForwardRef(Drawer))>
    </Hidden>
    <Hidden
      implementation="css"
      xsDown={true}
    >
      <WithStyles(ForwardRef(Drawer))
        className="makeStyles-drawer-2"
        classes={
          Object {
            "paper": "makeStyles-drawerPaper-6",
          }
        }
        variant="permanent"
      >
        <div
          className="makeStyles-toolbar-5"
        />
        <div>
          <WithStyles(ForwardRef(List))
            className="makeStyles-list-11"
          >
            <Link
              to="/"
            >
              <WithStyles(ForwardRef(Button))
                className="makeStyles-menulink-12"
              >
                Home
              </WithStyles(ForwardRef(Button))>
            </Link>
            <Link
              to="/about"
            >
              <WithStyles(ForwardRef(Button))
                className="makeStyles-menulink-12"
              >
                 About
              </WithStyles(ForwardRef(Button))>
            </Link>
          </WithStyles(ForwardRef(List))>
        </div>
      </WithStyles(ForwardRef(Drawer))>
    </Hidden>
  </nav>
</div>
`;
